src/obis-observer/commands/uplink/GetArchiveStateResponse.ts

```diff
@@ -0,0 +1,114 @@
+import Command, {TCommandExampleList} from '../../Command.js';
+import CommandBinaryBuffer, {REQUEST_ID_SIZE, ICommandParameters, DATE_TIME_SIZE} from '../../CommandBinaryBuffer.js';
+import {UPLINK} from '../../constants/directions.js';
+import {TTime2000} from '../../../utils/time.js';
+
+
+/**
+ * IGetArchiveStateResponseParameters command parameters
+ */
+interface IGetArchiveStateResponseParameters extends ICommandParameters {
+    archiveRecordsNumber: number,
+    eldestTime2000: TTime2000,
+    newestTime2000: TTime2000
+}
+
+const COMMAND_ID = 0x29;
+// request id byte + records count + DateTime 4 bytes * 2
+const COMMAND_SIZE = REQUEST_ID_SIZE + 4 + DATE_TIME_SIZE * 2;
+
+const examples: TCommandExampleList = [
+    {
+        name: 'no archive records',
+        parameters: {
+            requestId: 2,
+            archiveRecordsNumber: 0,
+            eldestTime2000: 0,
+            newestTime2000: 0
+        },
+        hex: {header: '29', body: '02 00 00 00 00 00 00 00 00 00 00 00 00'}
+    },
+    {
+        name: '81 records from 2023.06.27 18:45:02 GMT to 2023.06.28 15:15:02 GMT',
+        parameters: {
+            requestId: 2,
+            archiveRecordsNumber: 81,
+            eldestTime2000: 741206702,
+            newestTime2000: 741280502
+        },
+        hex: {header: '29', body: '02 00 00 00 51 2c 2d ea ae 2c 2f 0a f6'}
+    }
+];
+
+
+/**
+ * Uplink command.
+ *
+ * @example create command instance from command body hex dump
+ * ```js
+ * import GetArchiveStateResponse from 'jooby-codec/obis-observer/commands/uplink/GetArchiveStateResponse.js';
+ *
+ * const commandBody = new Uint8Array([
+ *     0x10, 0x22, 0x02, 0x2d, 0x19, 0x17, 0xc0, 0x32, 0x41, 0xb2, 0x28, 0xf6, 0x38, 0x42, 0xb2, 0xa8, 0xf6
```

Comment:
bad example. bytes and parameters not for this command

Reply:
fixed